modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -174,15 +174,18 @@ function acquia_cms_site_studio_rebuild_styles() {
  * Implements hook_content_model_role_presave_alter().
  */
 function acquia_cms_site_studio_content_model_role_presave_alter(RoleInterface &$role) {
-  if (in_array($role->id(), [
-    'content_administrator',
-    'content_editor',
-    'content_author',
-    'site_builder',
-  ])) {
-    $permissions = SiteStudioPermissionHelper::getSiteStudioPermissionsByRole($role->id());
-    foreach ($permissions as $permission) {
-      $role->grantPermission($permission);
+  $config = \Drupal::configFactory()->getEditable('cohesion_sync.cohesion_sync_package');
+  if ($config) {
```

Comment:
@deepakmishra2  can we do something like this?
if ($config = \Drupal::configFactory()->getEditable('cohesion_sync.cohesion_sync_package')) {

Reply:
updated these changes